types/snowflake-sdk/index.d.ts

```diff
@@ -86,8 +87,87 @@ export interface ConnectionOptions {
     jsTreatIntegerAsBigInt?: boolean;
 }
 
+export interface Column {
+    /**
+     * Returns the name of this column.
+     */
+    getName(): string;
+    /**
+     * Returns the index of this column.
+     */
+    getIndex(): number;
+    /**
+     * Returns the id of this column.
+     */
+    getId(): number;
+    /**
+     * Determines if this column is nullable.
+     */
+    isNullable(): boolean;
+    /**
+     * Returns the scale associated with this column.
+     */
+    getScale(): number;
+    /**
+     * Retuns the type associated with this column.
+     */
+    getType(): string;
+}
+
 export interface Statement {
+    /**
+     * Returns this statement's SQL text.
+     */
     getSqlText(): string;
+    /**
+     * Returns the current status of this statement.
+     * It will be either: {'fetching' or 'complete'}
+     */
+    getStatus(): string;
```

Comment:
This would be great as an enum, since it looks like [the source](https://github.com/snowflakedb/snowflake-connector-nodejs/blob/master/lib/connection/statement.js#L19-L23) is using an enum-like data structure.
```js
var states = {
  FETCHING: 'fetching',
  COMPLETE: 'complete'
};
```

Reply:
Sure. I have added the enum and the recommended spacing